test/unit/reflectivity_ui/interfaces/data_handling/test_dead_time_correction.py

```diff
@@ -0,0 +1,28 @@
+# package imports
+from reflectivity_ui.interfaces.data_handling.DeadTimeCorrection import SingleReadoutDeadTimeCorrection
+
+# 3rd-party imports
+import mantid.simpleapi as api
+import pytest
+from mantid.kernel import amend_config
+
+# standard imports
+
+
+@pytest.mark.datarepo
+@pytest.mark.parametrize("is_paralyzable", [False, True])
+def test_deadtime(is_paralyzable, data_server):
+    """Test of the dead-time correction algorithm SingleReadoutDeadTimeCorrection"""
+    with amend_config(data_dir=data_server.h5_full_path):
+        ws = api.Load("REF_M_42112")
+
+    algo = SingleReadoutDeadTimeCorrection()
```

Comment:
I'm puzzled. Why not use `mantid_algorithm_exec()` here instead of lines 19 to 24 :question: